modules/gallery/src/commonMain/kotlin/org/jetbrains/compose/storytale/gallery/story/StoryListItem.kt

```diff
@@ -29,36 +29,36 @@ import org.jetbrains.compose.storytale.gallery.ui.theme.currentColorScheme
 fun StoryListItem(
   story: Story,
   selected: Boolean,
+  onClick: () -> Unit,
```

Comment:
I'm wondering what rule caused this change :) 

Usually the trailing lambda indeed is the last parameter. But not in this case for some reason?